### DISCUSSION TITLE
Enable `Issues` in the `hedera-improvement-proposal` repo

### DISCUSSION THREAD

**Discussion by sergmetelin** [hiero-ledger/hiero-improvement-proposals#274] [General]
One of the things we are discussing is related to how the HIP feedback is tracked once it's merged into the HIPs repo. 

When a new HIP is created, the HIP author must also create a new Discussion thread. 

Community members then go to this HIP-specific Discussions thread on Github and leave comments/feedback. 
It's great to have a discussion there, but it's pretty difficult to track the material changes requested. 

We are thinking of enabling Github `Issues` functionality on the HIPs repo to request any changes through an issue. If issues related to a HIP aren't resolved by the `last-call-date-time`, HIP can't move to the next stage.

What are your thoughts on this?

**Comment by craigdrabiktxmq** [comment]
I think this is a good idea.  Discussions can get to be hard to follow and it's easier to relate issues to changes/pull requests to see what changed as a result of a discussion.
